include/nbl/builtin/hlsl/spirv_intrinsics/glsl.std.450.hlsl

```diff
@@ -18,13 +18,11 @@ template<typename Integral>
 [[vk::ext_instruction(GLSLstd450::GLSLstd450FindILsb, "GLSL.std.450")]]
 enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findILsb(Integral value);
 
-template<typename Integral>
 [[vk::ext_instruction(GLSLstd450::GLSLstd450FindSMsb, "GLSL.std.450")]]
-enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findSMsb(Integral value);
+int32_t findSMsb(int32_t value);
 
-template<typename Integral>
 [[vk::ext_instruction(GLSLstd450::GLSLstd450FindUMsb, "GLSL.std.450")]]
-enable_if_t<is_integral_v<Integral> && (sizeof(scalar_type_t<Integral>) == 4), Integral> findUMsb(Integral value);
+uint32_t findUMsb(uint32_t value);
```

Comment:
check if `findUMsb` doesn't return -1 for 0, if so then it needs to return `int32_t` despite taking in `uint32_t`

Reply:
also check `findILsb` while you're at it, i think it might return `int32_t` only as well